docs/advanced.rst

```diff
@@ -102,8 +102,8 @@ Also useful for deployment is the ``--deploy`` flag::
 This will fail a build if the ``Pipfile.lock`` is out–of–date, instead of generating a new one.
 
 
-☤ ``pipenv`` and ``conda``
---------------------------
+☤ Pipenv and Conda
+------------------
 
 To use Pipenv with a Conda–provided Python, you simply provide the path to the Python binary::
```

Comment:
This might need some more modifications. [Conda](https://conda.io) is a package manager + environment manager, and does not provide Python distributions. Anaconda does.

Reply:
Conda means both anaconda and miniconda?

Reply:
I don’t have an official reference from Continuum, but [Jake VanderPlas makes a clear distinction](http://jakevdp.github.io/blog/2016/08/25/conda-myths-and-misconceptions/#Myth-#1:-Conda-is-a-distribution,-not-a-package-manager).